controllers/semeru_compiler.go

```diff
@@ -345,6 +454,7 @@ func getLabels(wlva *wlv1.WebSphereLibertyApplication) map[string]string {
 	requiredLabels["app.kubernetes.io/managed-by"] = OperatorName
```

Comment:
The label `"app.kubernetes.io/instance"` must be unique since it is used in the label selector for Deployment and Service, so it should include the generation ID. This will avoid the new JIT server service from targeting old JIT server pods. 
The label `"app.kubernetes.io/name"` doesn't have to be unique, so it can remain as is.

Reply:
Done